applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D4N.hpp

```diff
@@ -372,6 +372,13 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ShellThickElement3D4N
 
     ///@}
 
+    ///@name Member Variables
+    ///@{
+

```

Comment:
pointer?

Reply:
After consultation with @philbucher we changed the member variable 'mpCoordinateTransformation' from private to protected because it is needed in the 'Create'-function of the derived adjoint elements. This was done in all shell element classes in the structual mechanics application.